app/helpers/value_helper.rb

```diff
@@ -1,17 +1,6 @@
 # frozen_string_literal: true
 
 module ValueHelper
-  # Renderers
-
-  # Given a fedora object, fetch the preferred object title from MODs, if not found, notify Honeybadger and return Fedora label
-  # @return [String] object title, as pulled from descMetadata
-  def object_title(obj)
-    return obj.descMetadata.title_info.first.strip if obj.descMetadata.title_info.first.present?
-
-    Honeybadger.notify("No title found in descMetadata for collection #{obj.pid}")
```

Comment:
The last occurance of this was two years ago, so it's not something we're hitting anymore: https://app.honeybadger.io/projects/49894/faults/45409599